eva-server/src/test/java/uk/ac/ebi/eva/server/UtilsTest.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.ebi.eva.server;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Tom Smith
+ */
+public class UtilsTest {
```

Comment:
If the API to Mongo mapping is extracted to `getDatabaseFieldMapping`, then 4 new test methods can be added here to check that all 3 fields are correctly mapped, and the rest do not have a mapping.